corehq/apps/reports/views.py

```diff
@@ -1012,6 +1067,7 @@ def page_context(self):
             'is_configurable_map': is_configurable_map,
             'languages_map': languages_map,
             'languages_for_select': languages_for_select,
+            'is_owner': self.request.couch_user._id == self.owner_id,
```

Comment:
This is a bug right now, should also include self.is_new (the way it is now, people cannot make new scheduled reports). This will be fixed in the next commit.